src/read/archive.rs

```diff
@@ -184,17 +226,60 @@ impl<'data, R: ReadRef<'data>> Iterator for ArchiveMemberIterator<'data, R> {
 /// A partially parsed archive member.
 #[derive(Debug)]
 pub struct ArchiveMember<'data> {
-    header: &'data archive::Header,
+    header: archive::MemberHeader,
     name: &'data [u8],
     offset: u64,
     size: u64,
 }
 
 impl<'data> ArchiveMember<'data> {
-    /// Parse the archive member header, name, and file data.
-    ///
-    /// This reads the extended name (if any) and adjusts the file size.
-    fn parse<R: ReadRef<'data>>(
+    /// Parse with AIX big archive style.
+    fn parse_aixbig<R: ReadRef<'data>>(
```

Comment:
Please reorder these three functions to be in the following order: `parse`, `parse_systemv`, `parse_aixbig`. This will also reduce the diff size.